### ISSUE TITLE
Highlight more products

### ISSUE THREAD

**Issue by arlin** [arlin/hip_hack_howto#36]
We have a project_products table (team outcomes) and an event_products table (event outcomes).  We need to be citing more of this, providing concrete examples of outcomes.  See issue #13.

**Comment by arlin** [comment]
Also, I'm a bit worried about the idea of proclaiming the effectiveness of NESCent hackathon when our most significant outcomes are papers, posters, and grant proposals.   That works in academica, but what about the rest of the world?   I suppose getting a grant is like getting investor buy-in for an idea in the private sector.   Maybe that is good enough.

**Comment by hlapp** [comment]
I don't think we have grounds to claim that the NESCent model would be effective outside of academia (and I actually doubt it would).

**Comment by rvosa** [comment]
Agreed. I don't think just focusing on scientific programming is too narrow
a scope at all considering that a lot of computational research domains are
having hackathons already or easily could.

Op Sun, 18 Oct 2015 om 04:55 schreef Hilmar Lapp notifications@github.com

**Comment by arlin** [comment]
The point I was trying to make is different.  We can count up posters and papers and proposals, and that is all fine and good, but I don't think that is making the best case for effectiveness.  Currently the manuscript expounds at length on how difficult it is to measure productivity and impact, but we haven't really gone to the trouble of even trying to assess productivity and impact in terms of information products (code and docs).   We can't apply robust metrics systematically to everything, but in the scale of quality for evidence, there are a lot of intermediate steps on the way to having robust metrics applied systematically.  The first step is just to pick some cases and study them.  For instance, I was on a team that developed a bunch of test files for NEXUS parsers and wrote a kind of compliance guide.  These were used in Bio::NEXUS and NCL.  How many test files did we produce at the hackathon?  Subsequently, were they used or referenced anywhere else?  Rutger was on a team that developed Java library support for NeXML.  I'd like to know if that was integrated into anyone's production system.  Obviously we can't do this for 54 projects.  That would take months of full-time effort.  But one of our goals is to lay the foundation for future research-- maybe our paper will convince someone to actually track down the impacts of 54 NESCent hackathon projects.

**Comment by rvosa** [comment]
Ah, I see. So do you think it makes sense to pick a couple of hackathon projects a write little stories about their impact beyond the hackathon? It would be anecdata - but would it go a small way to addressing what you're getting at?

**Comment by arlin** [comment]
Yes, I think it makes sense.   We could pick a random sample of 5 teams, and ask just a few questions.  What was the main tangible product (a prototype, new code added to existing codebase, a draft standard, a design)?  How is the product being used now?   Did the product lead (directly or indirectly) to tangible impacts?

**Comment by arlin** [comment]
Also, if we really want to stimulate followup and lay a foundation, why not develop a set of carefully framed questions, apply them to 5 randomly chosen teams,  and then report on how many hours it took us to do that?   That makes it easier for some professor out there to make a decision about whether to put a graduate student onto this.

**Comment by arlin** [comment]
Here is my attempt at a relatively short form to cover products and impacts: https://docs.google.com/forms/d/1czuQxzfx7CsDeMIRUk8F4tbUsrou2Lw4f1g8srWH5r8/viewform?usp=send_form

**Comment by arlin** [comment]
OK, this deserves a separate ticket.  This ticket is about adding more information on products to the manuscript.  The separate ticket will be about gathering better information on a sample of products.

**Comment by arlin** [comment]
This is blocked by #48.  We have to do the research, then we can write the results into the manuscript.

**Comment by rvosa** [comment]
Given that #48 is now closed and the manuscript is littered with references to specific projects, I think this issues has become moot.
